pyproject.toml

```diff
@@ -62,4 +51,53 @@ show_column_numbers = true
 mypy_path = "../.stubs"
 # Exclude generated files
 # TODO Remove when we drop python 2
-exclude = '.*/config_models/.*\.py$'
\ No newline at end of file
+exclude = '.*/config_models/.*\.py$'
+
+[tool.ruff]
+exclude = [
+    ".eggs",
+    ".tox",
+    "build",
+    "compat.py",
+    "__init__.py",
+    "**/datadog_checks/dev/tooling/templates/*",
+    "**/datadog_checks/*/vendor/*",
+]
+target-version = "py37"
```

Comment:
Why not `py38`?

Reply:
Not really sure. I had ported this config over from the hatch template that Ofek recommended:
https://github.com/pypa/hatch/blob/master/src/hatch/template/files_default.py#L187

Reply:
yes